### ISSUE TITLE
[server] Allow editing a given drawing

### ISSUE THREAD

**Issue by zaidalyafeai** [ARBML/Calliar#8]

**Comment by zaidalyafeai** [comment]
![unknown-4](https://user-images.githubusercontent.com/15667714/188617497-ab55c569-2a1d-4c72-998d-913b891b1575.png)

**Comment by Shubbair** [comment]
Do you need this design to be on the website?

**Comment by zaidalyafeai** [comment]
Yes on the main page. If you have any questions let me know. The basic idea is to allow the user to edit a stroke that was written in the past. The main js script is paper.js.

**Comment by Shubbair** [comment]
Okay, I will try to do it in an efficient way.

**Comment by Shubbair** [comment]
Zaid, I just wonder why you need to make a drag option to the letter ? 
I think clear and update is enough because it's already in the correct sequence.
